FLTextView/FLTextView.swift

```diff
@@ -339,6 +339,9 @@ extension FLTextView: UITextViewDelegate {
                     placeholderView.resignFirstResponder()
                     self.text = frozenText + text
                     becomeFirstResponder()
+                    if self.isExternalTextViewDelegateRespondsToSelector("textViewDidChange:") {
+                        self.externalTextViewDelegate!.textViewDidChange!(self)
```

Comment:
probably make sense to have it in view 
externalTextViewDelegate?.textViewDidChange?(self)
to be more safety